source/elements/oneMKL/source/architecture/api_design.inc.rst

```diff
@@ -34,18 +35,18 @@ oneMKL uses C++ STL data types for scalars where applicable:
 * Integer scalars are C++ fixed-size integer types (``std::intN_t``, ``std::uintN_t``).
 * Complex numbers are represented by C++ ``std::complex`` types.
 
-In general, scalar integer arguments to oneMKL routines are 64 bit integers (``std::int64_t`` or ``std::uint64_t``). Integer vectors and matrices may have varying bit widths, defined on a per-routine basis.
+In general, scalar integer arguments to oneMKL routines are 64-bit integers (``std::int64_t`` or ``std::uint64_t``). Integer vectors and matrices may have varying bit widths, defined on a per-routine basis.
 
 .. _onemkl_dpcpp_datatypes:
 
 DPC++ datatype usage
 ++++++++++++++++++++
 
-oneMKL uses following SYCL Language data types and DPC++ Language Extensions data types:
+oneMKL uses following SYCL language data types and DPC++ language extensions data types:
 
 * SYCL queue ``sycl::queue`` for scheduling kernels on a SYCL device. See :ref:`onemkl_queues` for more details.
-* SYCL buffer ``sycl::buffer`` for buffer based memory access. See :ref:`onemkl_buffers` for more details.
-* DPC++ Extension Unified Shared Memory (USM) for pointer based memory access. See :ref:`onemkl_usm` for more details.
+* SYCL buffer ``sycl::buffer`` for buffer-based memory access. See :ref:`onemkl_buffers` for more details.
+* DPC++ extension Unified Shared Memory (USM) for pointer-based memory access. See :ref:`onemkl_usm` for more details.
```

Comment:
* Unified Shared Memory (USM) for pointer-based memory access. See :ref:`onemkl_usm` for more details.

Reply:
Updated